### DISCUSSION TITLE
Weird lighting behavior even with 2f ambient light

### DISCUSSION THREAD

**Discussion by GoldenCookieGIT** [Minestom/Minestom#1479] [Help]
https://gyazo.com/ea4523e78f2771ad1c676bf98ea0b8c8
![image](https://user-images.githubusercontent.com/45347578/197904877-0a10fbf0-f4d7-408f-85e3-0bbaff117202.png)
![image](https://user-images.githubusercontent.com/45347578/197904896-7506b8c0-f113-4d7b-943c-1af30905ff09.png)
![image](https://user-images.githubusercontent.com/45347578/197904948-dc26e3b8-2afc-4db0-b74f-2aad19283e2f.png)

weird lighting behavior even with 2f ambient light, my dimension gets registered and gets set to my instance correctly.

with the generic no lighting behavior the dark parts stay dark, but as you can see in the gif it's doing something weird.

**Comment by Tofaa2** [comment]
Have you tried resending the player the chunk data? Might fix your issue
